projects/components/src/lib/directives/show-clipped-text.directive.spec.ts

```diff
@@ -120,6 +121,18 @@ describe('ShowClippedTextDirective', () => {
         // });
     });
 
+    describe('disabling the tooltip', () => {
+        it('wont show a tooltip when disabled', fakeAsync(function(this: Test): void {
+            const helper = this.clippedTextHelper;
+            helper.disabled = true;
```

Comment:
I think the disabled setter should call detect changes, then we wouldn't need to expose the fixture and we'd avoid some duplication